projects/hello_world/src/main.c

```diff
@@ -0,0 +1,7 @@
+#include "log.h"
+
+int main(void) {
+  LOG_DEBUG("Hello World!\n");
+
+  return 0;
+}

```

Comment:
The `hello_world` project isn't part of the homework, so please don't commit it.